### ISSUE TITLE
imgurbash no longer works (and seems unlikely to get fixed?)

### ISSUE THREAD

**Issue by michalrus** [NixOS/nixpkgs#14444]
## Issue description

The Imgur API has changed.

See the discussion in https://aur.archlinux.org/packages/imgurbash/ .

### Steps to reproduce

```
% nix-shell -p scrot imgurbash --run 't=$(mktemp) ; scrot $t.png ; imgurbash $t.png'

Delete page:
% curl -Ssf http://imgur.com/api/upload.xml
curl: (22) The requested URL returned error: 404 Not Found
%
```

## Technical details

* System: NixOS 16.09pre79453.32b7b00 (Flounder)
* Nix version: nix-env (Nix) 1.11.2
* Nixpkgs version: "16.09pre79453.32b7b00"

**Comment by jgillich** [comment]
This is an upstream issue, right? Are you suggesting we remove it ("unlikely to get fixed")?

**Comment by michalrus** [comment]
@jgillich, totally upstream, yeah. However, there’s been no contact with the upstream developer so far.

Arch users are using this one now → https://github.com/ram-on/imgurbash2 (cf. https://aur.archlinux.org/packages/imgurbash/).

Should I make a PR for us to switch to it?

**Comment by michalrus** [comment]
No, wait, the original author has been summoned.

**Comment by abbradar** [comment]
I think of removing `imgurbash` from repositories and replacing it with `imgurbash2` (I like a GitHub project more than a patch from AUR maintainer -- at least we can report issues and have a better source URL). I don't have a strong opinion on this however; What do you think?

**Comment by abbradar** [comment]
Closed by 59bed14119a2842947e948efbd696c51616cb687

**Comment by abbradar** [comment]
For our release branch I've decided to patch `imgurbash` instead (to minimize amount of external changes). See 4b667665c2c459a02c8d37a5e97302a621fcfbb5.
